src/api/models/__init__.py

```diff
@@ -0,0 +1,9 @@
+# coding: utf-8
```

Comment:
You should have a standard docstring here:
```suggestion
"""The models subpackage."""
```

Also, since [the default encoding for Python 3 files is UTF-8](https://docs.python.org/3/howto/unicode.html#the-string-type), the `# coding: utf-8` line is unnecessary here and elsewhere.